### PR TITLE
Feature/no data messages

### PR DESCRIPTION
**Pull request by noahmanger**
This updates the "no data" messages from https://github.com/18F/openFEC-web-app/issues/1888

It also makes a few small improvements to candidate pages:
- Adds `entity__figure--narrow` class to set a max-width on entity figures, like:
image

- Rearranges the tabs to match the original designs

Still making final copy edits to the messages.

### CODE REVIEW

File: openfecwebapp/templates/macros/missing.html [hunk 1]
```diff
@@ -1,14 +1,15 @@
 {% macro missing_financials(entity, cycle) %}
-<div class="message message--no-icon">
-  <h2 class="message__title" id="section-1-header" tabindex="0">No results</h2>
-  <p>Sorry, we didn't find any financial data for <span class="t-bold">{{ entity }}</span> in the <span class="t-bold">{{ cycle | fmt_year_range }}</span> election cycle.</p>
+<div class="message message--info">
+  <p>We don't have any financial data for <span class="t-bold">{{ entity }}</span> in <span class="t-bold">{{ cycle | fmt_year_range }}</span>.</p>
+  <p>This can either be because:</p>
```
Comment (emileighoutlaw) [inline openfecwebapp/templates/macros/missing.html]: Now that I'm looking at this again, I'm worried that the "either" makes it seem like these are the only two possible reasons, which I don't think is the case. What if we said 

_Sometimes this happens when a candidate [or committee]:_ 

instead?

Comment (noahmanger) [inline openfecwebapp/templates/macros/missing.html]: That doesn't make sense with the next bullet being "There hasn't been a filing deadline yet" so what about just removing the "either"?

File: openfecwebapp/templates/candidates-single.html [hunk 2]
```diff
@@ -56,28 +56,28 @@ <h1 class="entity__name">{{ name }}</h1>
           <a
             class="side-nav__link"
             role="tab"
-            data-name="other-spending"
-            tabindex="2"
+            data-name="individual-contributions"
+            tabindex-"3"
```
Comment (xtine) [inline openfecwebapp/templates/candidates-single.html]: `-` instead of `=`

Comment (noahmanger) [inline openfecwebapp/templates/candidates-single.html]: Oh thanks

**Pull request merged**